src/Controls/src/Build.Tasks/NodeILExtensions.cs

```diff
@@ -157,6 +157,8 @@ static T TryFormat<T>(Func<string, T> func, IXmlLineInfo lineInfo, string str)
 					{ module.ImportReference(("Microsoft.Maui", "Microsoft.Maui.Converters", "EasingTypeConverter")), typeof(EasingTypeConverter) },
 					{ module.ImportReference(("Microsoft.Maui.Graphics", "Microsoft.Maui.Graphics.Converters", "ColorTypeConverter")), typeof(ColorTypeConverter) },
 					{ module.ImportReference(("Microsoft.Maui.Graphics", "Microsoft.Maui.Graphics.Converters", "PointTypeConverter")), typeof(PointTypeConverter) },
+					{ module.ImportReference(("Microsoft.Maui.Graphics", "Microsoft.Maui.Graphics.Converters", "RectTypeConverter")), typeof(RectangleTypeConverter) },
```

Comment:
what about this ? isn't it out of scope ? (I'm fine if you re-scope the PR by changing title and description)

Reply:
Are you saying you want this part split out into a second PR? In the PR description:
```
`RectTypeConverter` is missing the same treatment? I added an entry in
`KnownCompiledTypeConverters`, that seems like it was just missing?
```

Reply:
no need for a 2nd PR, but mention it in the description